Off-Top-App/off-top-flutter
# Ot 90

**What I did:**
- Sent an audio file into our spring boot server
- converted the audio file into bytes and am sending it through the WebSocket
- the goal here is to send an int list through the websocket so that java can have the file in byte array form (and possibly eventually convert it back to some sort of audio file).



**How to Test:**
1. When the flutter and spring-boot server is running make way to the WebSocket page. Click on the "send data" button to see data coming in. You will see a bunch of numbers appearing, that is fine because it is byte data of the audio file. 
2. Check the code and see if there can be any improvements.

 **Note:** You will need to checkout into OT-90 in the off-top-java repo to look at the incoming data from the WebSocket. You can view it here: https://github.com/Off-Top-App/off-top-java/pull/12
